spyder/plugins/variableexplorer/widgets/arrayeditor.py

```diff
@@ -847,8 +847,8 @@ def do_nothing():
             _('<u>Warning</u>: Changes are applied separately')
         )
         self.masked_label.setToolTip(
-            _("For performance reasons, changes applied to masked arrays won't"
-              "be reflected in array's data (and vice-versa).")
+            _("For performance reasons, changes applied to masked arrays "
+              "won't be reflected in array's data (and vice-versa).")
           )
```

Comment:
```suggestion
        )
```

To align this parenthesis with the text that opens it.